lib/Parser/Hash.h

```diff
@@ -266,8 +267,16 @@ struct Ident
                     // Without parameter scope, we would have just pushed the ref instead of inserting.
                     // We effectively had a false positive match (a parameter scope ref with no sym)
                     // so we need to push the Pid rather than inserting.
```

Comment:
I believe this comment required some update. As we are inserting in between two positions.